Zend/zend_ast.c

```diff
@@ -776,12 +776,22 @@ ZEND_API zend_result ZEND_FASTCALL zend_ast_evaluate(zval *result, zend_ast *ast
 			zend_string *case_name = zend_ast_get_str(case_name_ast);
 
 			zend_ast *case_value_ast = ast->child[2];
-			zval *case_value_zv = case_value_ast != NULL
-				? zend_ast_get_zval(case_value_ast)
-				: NULL;
+
+			zval case_value_zv;
+			zval *case_value_zv_ptr = NULL;
```

Comment:
I'd consider dropping case_value_zv_ptr here and use an UNDEF zval instead. Can dtor unconditionally below.